content/general/events/azure.textile

```diff
@@ -0,0 +1,45 @@
+---
+title: Azure Functions
+section: general
+index: 23
+languages:
+  - none
+jump_to:
+  Help with:
+    - Fields#fields
+    - Example#example
+---
+
+As part of "Reactor Events":/general/events, it is possible to integrate with various systems.
+
+"Azure Functions":https://azure.microsoft.com/en-gb/services/functions provide event-driven serverless compute functions which allow users to easily run code whenever events are sent to it.
```

Comment:
Should we not somewhere indicate that all the detail that sits behind how Reactor Events work, is in the Reactor Events docs.  I am concerned someone arriving here, won't realise that.